dt/uni07eta/kustomization.yaml

```diff
@@ -344,13 +340,14 @@ replacements:
         options:
           create: true
 
-  - source:
-      kind: ConfigMap
-      name: network-values
-      fieldPath: data.octavia.net-attach-def
-    targets:
-      - select:
-          kind: NetworkAttachmentDefinition
-          name: octavia
-        fieldPaths:
-          - spec.config
+# # TODO: check this again
+# - source:
+#     kind: ConfigMap
+#     name: network-values
+#     fieldPath: data.octavia.net-attach-def
+#   targets:
+#     - select:
+#         kind: NetworkAttachmentDefinition
+#         name: octavia
+#       fieldPaths:
+#         - spec.config
```

Comment:
Should be able to drop this since it is moved to `dt/uni07eta/networking/kustomization.yaml` now

Reply:
@dsariel Maybe we can remove this commented section since it doesn't seem to be needed?  Otherwise, given that testing is done, I think we can merge this.

Reply:
@dsariel If you could remove this commented section, we could merge this